PancakeSwap-Bots/Set-Masterchef-Settings/src/agent.spec.ts

```diff
@@ -0,0 +1,230 @@
+import { Interface } from "ethers/lib/utils";
+import { FindingType, FindingSeverity, Finding, HandleTransaction, TransactionEvent, Network } from "forta-agent";
+import { TestTransactionEvent } from "forta-agent-tools/lib/test";
+import { createAddress } from "forta-agent-tools";
+import { provideHandleTransaction } from "./agent";
+import { BigNumber } from "ethers";
+import { NetworkManager } from "forta-agent-tools";
+import { NetworkData } from "./network";
+import abi from "./abi";
+
+const MOCK_ABI: string[] = ["function mockFunction(address mockAddress)"];
+
+const testMasterchef: string = createAddress("0x01");
+
+const DEFAULT_CONFIG: Record<number, NetworkData> = {
+  [Network.MAINNET]: {
```

Comment:
`Network.MAINNET` is Ethereum network which means chainId is 1. it might not matter since it's testing but it would still be better if it was `Network.BSC`